active/Breadcrumb/Breadcrumb.md

```diff
@@ -0,0 +1,225 @@
+### Examples
+
+The following example code creates a simple Breadcrumb control with three nodes.
+When there is not enough width to display the whole control,
+the leading nodes are replaced with a node that the user can click on,
+opening the missing nodes in a flyout menu.
+
+![Breadcrumb with nodes: Home, Documents, Design, Northwind, Images. The app is resized so that the Breadcrumb crumbles - then, clicking the ellipsis reveals the crumbled Home and Documents nodes](images/Breadcrumb_crumbling_flyout.gif)
+
+XAML
+```XAML
+<Breadcrumb x:Name="Breadcrumb1" /> 
+```
+
+C#
+```cs
+Breadcrumb1.ItemsSource = new string[]{ "Home", "Documents", "Design", "Northwind", "Images"}
+```
+
+If the last item in a Breadcrumb is text and is too long to fit it will be clipped
+(if the item type is a string it will be trimmed, meaning shortened and replaced with an ellipsis).
+
+![Breadcrumb_crumbled with last node truncated](images/Breadcrumb_truncation.PNG)
+
+#### Responding to clicks
+
+This example shows an app with a Breadcrumb across the top, and a content area on the rest of the Window.
+The Breadcrumb shows a list of items. Clicking on an item navigates the content back, and updates
+the Breadcrumb.
+
+```xaml
+<Grid RowDefinitions="Auto,*">
+    <Breadcrumb ItemClicked="CrumbClicked" ItemsSource="{x:Bind Items, Mode=OneWay}" />
+    
+    <ContentControl Grid.Row="1" Content="{x:Bind CurrentItem, Mode=OneWay}">
+        ...
+    </ContentControl>
+</Grid>
+```
+
+```csharp
+public sealed partial class MainPage : Page
+{
+    public ObservableCollection<Items> Items { get; }
+
+    public ItemClickEventArgs Current Item { get; set;}
+
+    void CrumbClicked(Breadcrumb sender, BreadcrumbItemClickedEventArgs e)
+    {
+        for(int i = Items.Count-1; i > e.Index; i++)
+        {
+            Items.RemoveAt(i);
+        }
+
+        CurrentItem = Items[i - 1];
+    }
+
+    void NavigateToItem(Item item)
+    {
+        Items.Add(item);
+        CurrentItem = item;
+    }
+}
+```
+
+#### Breadcrumb.ItemTemplate property
+
+Gets or sets the template to display an item
+
+The Breadcrumb ItemTemplate is used to override the default display of the
+individual items in the Breadcrumb control.
+
+The following uses the ItemTemplate so that the Breadcrumb items show the Name property of the items:
+
+```xaml
+<Breadcrumb ItemsSource='{x:Bind Items}'>
+    <Breadcrumb.ItemTemplate>
+        <DataTemplate x:DataType='BreadcrumbItem`>
+            <TextBlock Text='{x:Bind Name}/>
+        </DataTemplate>
+    </Breadcrumb.ItemTemplate>
+</Breadcrumb>
+```
+
+Notes:
+* You can use a BreadcrumbItem as the root of the DataTemplate
+* If no ItemTemplate is supplied, the Breadcrumb control will convert to a string.
+
+## Other Breadcrumb members
+
+| Name | Description | Default |
+| :---------- | :------- | :------- |
+| ItemsSource | Gets or sets the content of the Breadcrumb |
+| ItemClicked | Raised when a user interaction causes a jump to the CurrentItem.  |
```

Comment:
As the content of this control is set via DataBinding, it seems at odds that there is also no `ICommand` built in to handle the "Click" event.  To be more MVVM-friendly, it would be great if there wasn't a need to use an EventToCommand conversion.

Add a `Command` member that can also be invoked when an item is "clicked" so as to match the behavior of other controls and provide a consistent API experience.